README.es-ES.md

```
@@ -0,0 +1,308 @@
+
+### Consultas Avanzadas
+
+```
+"Busca todos los productos electrónicos que estén en stock y cuesten menos de $1000"
+"Muéstrame todos los pedidos del usuario john@example.com"
+"Lista los productos con calificaciones superiores a 4.5"
+```
+
+### Gestión de Índices
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)